src/Nodes/Texts/SVGText.php

```diff
@@ -1,13 +1,83 @@
 <?php
-
 namespace SVG\Nodes\Texts;
 
+use SVG\Nodes\Structures\SVGFont;
 use SVG\Nodes\SVGNodeContainer;
+use SVG\Rasterization\SVGRasterizer;
 
 /**
  * Represents the SVG tag 'text'.
+ *
+ * Usage:
+ *
+ * $svg = new \SVG\SVGImage(600, 400);
+ *
+ * $font = new \SVG\Nodes\Structures\SVGFont('openGost', 'OpenGostTypeA-Regular.ttf');
+ * $svg->getDocument()->addChild($font);
+ *
+ * $svg->getDocument()->addChild(
+ *   (new \SVG\Nodes\Texts\SVGText('hello', 50, 50))
+ *     ->setFont($font)
+ *     ->setSize(15)
+ *     ->setStyle('stroke', '#f00')
+ *     ->setStyle('stroke-width', 1)
+ * );
+ *
  */
 class SVGText extends SVGNodeContainer
 {
     const TAG_NAME = 'text';
+
+    /**
+     * @var SVGFont
+     */
+    private $font;
+
+    public function __construct($text = '', $x = 0, $y = 0)
+    {
+        parent::__construct();
+        $this->setValue($text);
+        if($x > 0) {
+            $this->setAttribute('x', $x);
+        }
+        if($y > 0) {
+            $this->setAttribute('y', $y);
+        }
+    }
+
+    public function rasterize(SVGRasterizer $rasterizer)
+    {
+        $rasterizer->render('text', array(
+            'x'         => $this->getAttribute('x'),
+            'y'         => $this->getAttribute('y'),
+            'size'      => intval($this->getComputedStyle('font-size')),
```

Comment:
Just calling `intval` might be a bit too easy. Font size can be specified in all kinds of units.

Reply:
In addition, intval is a PHP4 function. Typecasting would be more performant in this case.